plugins/org.yakindu.sct.generator.cpp/src/org/yakindu/sct/generator/cpp/CppKeywords.java

```diff
@@ -13,6 +13,5 @@
 
 public final class CppKeywords {
 	
-	public static final String[] CPP_KEYWORDS = { "clearInEvents", "clearOutEvents", "enter",
-		"exit", "runCycle", "parent"};
+	public static final String[] CPP_KEYWORDS = {"parent"};
```

Comment:
Why are enter, exit, runcycle no more marked as keywords? The methods will still be generated.

Reply:
They are now part of the Sequencer. The NamingService detects them and changes the ``runCycle`` method to ``runCycle_ID``.